phpunit.xml.dist

```diff
@@ -6,10 +6,6 @@
          colors="true"
          bootstrap="tests/bootstrap.php"
 >
-    <php>
-        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0" />
```

Comment:
This is not a good idea as the build will break as soon as a third party library deprecates something triggered by our own code. This is why this line was added. Please leave it as is or check the [documentation](https://symfony.com/doc/current/components/phpunit_bridge.html#internal-deprecations) to see if there is a more appropriate configuration that we can use

Reply:
We can try `max[direct]=0`, but I have to check if it detects this kind of issues.

Reply:
`max[self]` worked as far as I see from build logs, it was reporting 3 legacy deprecations, thus I don’t get what you are trying to fix by changing/removing this setting

Reply:
It didn't make the build fail, hence I had to do this late PR.

Reply:
It did’t make the build fail on purpose, as doing it would move the burden of fixing the issue to the next person that will contribute a change, and if no one do it we will never know anyway that we are generating deprecation notices...

Reply:
IMHO we should let the build fail instead. We can always open a parallel PR to fix the deprecations, and ignore them as a last resort when merging.

Reply:
As I wrote above until someone opens a PR and gets the failing build in the face we won’t know that something is generating deprecation notices, so I don’t really see how removing this line will help us to not ignore/miss such notices. We should periodically run a build just to check that in the meanwhile another library didn’t deprecate something we rely on, so this change is kinda useless in my opinion. The fact is that in the rush of releasing a version compatible with PHP 7.4 we forgot to check if any deprecation was there. To clarify, if no change was required to release such version we would have missed the deprecations too